src/index.html

```diff
@@ -17,11 +18,360 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="./index.scss"
```

Comment:
The stylesheet is linked as `index.scss`, which is not directly usable by browsers. Ensure that this SCSS file is compiled to CSS and the correct CSS file is linked here.